tests/console/cups.pm

```diff
@@ -68,8 +68,8 @@ sub run {
     record_info "lpadmin -x", "Removing printers";
     assert_script_run "lpadmin -x $_" foreach (qw/printer_tmp printer_null/);
     validate_script_output 'lpstat -p -d -o 2>&1 || test $? -eq 1', sub { m/No destinations added/ };
-    systemctl 'stop cups.service';
     systemctl 'disable cups.service';
+    systemctl 'stop cups.service';
```

Comment:
Maybe `systemctl 'disable --now cups'` would also work (if this test does not run on old systems)

Reply:
Be careful, I personally had issues with `disable --now`, disable was ok, but not the 'now' part...

Reply:
`disable --now` is certainly supported in openSUSE Tumbleweed but not in older product versions that we still want to support